packages/client/test/end-to-end/publish-subscribe.test.ts

```diff
@@ -1,29 +1,33 @@
-import { Wallet } from '@ethersproject/wallet'
-import { StreamID, toStreamPartID, TrackerRegistryRecord } from '@streamr/protocol'
-import { createNetworkNode, NetworkNodeOptions, TEST_CONFIG } from '@streamr/network-node'
+import { Wallet } from 'ethers'
+import { StreamID, toStreamPartID } from '@streamr/protocol'
 import { fastWallet, fetchPrivateKeyWithGas } from '@streamr/test-utils'
 import { CONFIG_TEST } from '../../src/ConfigTest'
 import { PermissionAssignment, StreamPermission } from '../../src/permission'
 import { Stream } from '../../src/Stream'
 import { StreamrClient } from '../../src/StreamrClient'
-import { createTestStream } from '../test-utils/utils'
-import { waitForCondition, MetricsContext } from '@streamr/utils'
+import { entryPointTranslator } from '../../src/utils/utils'
+import { createTestStream, createTestClient } from '../test-utils/utils'
+import { waitForCondition } from '@streamr/utils'
+import { NetworkNode } from '@streamr/trackerless-network'
 
-const TIMEOUT = 20 * 1000
+const TIMEOUT = 30 * 1000
 
 const PAYLOAD = { hello: 'world' }
 
 const ENCRYPTED_MESSSAGE_FORMAT = /^[0-9A-Fa-f]+$/
 
 async function startNetworkNodeAndListenForAtLeastOneMessage(streamId: StreamID): Promise<unknown[]> {
-    const config: NetworkNodeOptions = {
-        ...TEST_CONFIG,
-        id: 'networkNode',
-        trackers: CONFIG_TEST.network!.trackers as TrackerRegistryRecord[],
-        metricsContext: new MetricsContext()
-    }
-    const networkNode = createNetworkNode(config)
+    const entryPoints = entryPointTranslator(CONFIG_TEST.network!.layer0!.entryPoints!)
+    const networkNode = new NetworkNode({
+        ...CONFIG_TEST.network as any,
+        layer0: {
+            entryPoints,
+        },
+        networkNode: {} 
```

Comment:
The `networkNode` field of `NetworkOptions` could be optional as all fields of `StreamrNodeConfig` are optional

Reply:
The values here are given directly to the NetworkNode as it is started directly here. In this case it should be okay to assign an empty value

Reply:
Is there a reason to keep it as required? If we don't need to define anything about the node, it would be simpler just to leave out the config block.